File: src/main/resources/output/IFC4X3_csharp/Entities/IfcProfileDef/IfcParameterizedProfileDef/IfcLShapeProfileDef/IfcLShapeProfileDef.cs
public class IfcLShapeProfileDef : IfcParameterizedProfileDef
{
    public IfcPositiveLengthMeasure Depth { get; set; }
    public IfcPositiveLengthMeasure Width { get; set; }
    public IfcPositiveLengthMeasure Thickness { get; set; }
    public IfcNonNegativeLengthMeasure FilletRadius { get; set; }
    public IfcNonNegativeLengthMeasure EdgeRadius { get; set; }
    public IfcPlaneAngleMeasure LegSlope { get; set; }

    // === WHERE CLAUSES ===
    // ValidThickness : (Thickness < Depth) AND (NOT(EXISTS(Width)) OR (Thickness < Width))
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcProfileDef/IfcParameterizedProfileDef/IfcLShapeProfileDef/IfcLShapeProfileDef.py
class IfcLShapeProfileDef(IfcParameterizedProfileDef):
    def __init__(self):
        self.Depth: IfcPositiveLengthMeasure = None
        self.Width: IfcPositiveLengthMeasure = None
        self.Thickness: IfcPositiveLengthMeasure = None
        self.FilletRadius: IfcNonNegativeLengthMeasure = None
        self.EdgeRadius: IfcNonNegativeLengthMeasure = None
        self.LegSlope: IfcPlaneAngleMeasure = None

    # === WHERE CLAUSES ===
    # ValidThickness : (Thickness < Depth) AND (NOT(EXISTS(Width)) OR (Thickness < Width))
